<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-KVHQD5O" failures="0" tests="2" name="com.internetbanking.testcases.TC_LoginDDT_002" time="-1743163520.000" errors="2" timestamp="2025-03-28T17:33:50 IST" skipped="0">
  <testcase classname="com.internetbanking.testcases.TC_LoginDDT_002" name="setup" time="-1743163520.000">
    <error message="
Parameter &#039;browser&#039; is required by BeforeClass on method setup but has not been marked @Optional or defined
in C:\Users\v\AppData\Local\Temp\testng-eclipse-1632749446\testng-customsuite.xml" type="org.testng.TestNGException">
      <![CDATA[org.testng.TestNGException: 
Parameter 'browser' is required by BeforeClass on method setup but has not been marked @Optional or defined
in C:\Users\v\AppData\Local\Temp\testng-eclipse-1632749446\testng-customsuite.xml
at org.testng.internal.Parameters.createParams(Parameters.java:264)
at org.testng.internal.Parameters.createParametersForMethod(Parameters.java:352)
at org.testng.internal.Parameters.createParameters(Parameters.java:690)
at org.testng.internal.Parameters.createConfigurationParameters(Parameters.java:163)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:313)
at org.testng.internal.invokers.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:188)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </error>
  </testcase> <!-- setup -->
  <testcase classname="com.internetbanking.testcases.TC_LoginDDT_002" name="loginDDT" time="0.000">
    <error message="java.io.FileNotFoundException: null\src\test\java\com\internetbanking\testData\USER Data.xlsx (The system cannot find the path specified)" type="java.lang.RuntimeException">
      <![CDATA[java.lang.RuntimeException: java.io.FileNotFoundException: null\src\test\java\com\internetbanking\testData\USER Data.xlsx (The system cannot find the path specified)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:58)
at org.testng.internal.invokers.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:159)
at org.testng.internal.Parameters.handleParameters(Parameters.java:809)
at org.testng.internal.Parameters.handleParameters(Parameters.java:734)
at org.testng.internal.invokers.ParameterHandler.handleParameters(ParameterHandler.java:72)
at org.testng.internal.invokers.ParameterHandler.createParameters(ParameterHandler.java:51)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:949)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
Caused by: java.io.FileNotFoundException: null\src\test\java\com\internetbanking\testData\USER Data.xlsx (The system cannot find the path specified)
at java.base/java.io.FileInputStream.open0(Native Method)
at java.base/java.io.FileInputStream.open(FileInputStream.java:219)
at java.base/java.io.FileInputStream.<init>(FileInputStream.java:159)
at java.base/java.io.FileInputStream.<init>(FileInputStream.java:112)
at com.internetbanking.utilities.XLUtils.getRowCount(XLUtils.java:25)
at com.internetbanking.testcases.TC_LoginDDT_002.getData(TC_LoginDDT_002.java:64)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:54)
... 25 more
]]>
    </error>
  </testcase> <!-- loginDDT -->
</testsuite> <!-- com.internetbanking.testcases.TC_LoginDDT_002 -->
